pipelines/plain_pipelines/paas-hackmd.yml

```diff
@@ -131,6 +138,7 @@ jobs:
                     CMD_DOMAIN: 'hackmd.${CF_APPS_DOMAIN}'
                     CMD_PROTOCOL_USESSL: 'true'
                     CMD_ALLOW_ORIGIN: 'localhost,hackmd.${CF_APPS_DOMAIN},cdnjs.cloudflare.com,fonts.googleapis.com,fonts.gstatic.com'
```

Comment:
This doesn't match the current list in prod `localhost,hackmd.${CF_APPS_DOMAIN},cdnjs.cloudflare.com,fonts.googleapis.com,fonts.gstatic.com,githubusercontent.com,github.com`